services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from django.shortcuts import get_object_or_404
+
+import init_django_orm  # noqa: F401
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
```

Comment:
The function `create_user` is annotated to return `None`, but it might be beneficial to return the created `User` object for further use or confirmation.